src/ntfs/read.rs

```diff
@@ -186,14 +179,14 @@ pub fn manage_codes(collections: &mut Collections, path: &path::Path) {
     let mut rdr = csv::Reader::from_path(path.join("object_codes.txt")).unwrap();
     for code in rdr.deserialize().map(Result::unwrap) {
         let code: Code = code;
-        match code.object_type.as_str() {
-            "stop_area" => insert_code(&mut collections.stop_areas, code),
-            "stop_point" => insert_code(&mut collections.stop_points, code),
-            "network" => insert_code(&mut collections.networks, code),
-            "line" => insert_code(&mut collections.lines, code),
-            "route" => insert_code(&mut collections.routes, code),
-            "trip" => insert_code(&mut collections.vehicle_journeys, code),
-            _ => panic!("{} is not a valid object_type", code.object_type),
+        match code.object_type {
+            ObjectType::StopArea => insert_code(&mut collections.stop_areas, code),
+            ObjectType::StopPoint => insert_code(&mut collections.stop_points, code),
+            ObjectType::Network => insert_code(&mut collections.networks, code),
+            ObjectType::Line => insert_code(&mut collections.lines, code),
+            ObjectType::Route => insert_code(&mut collections.routes, code),
+            ObjectType::VehicleJourney => insert_code(&mut collections.vehicle_journeys, code),
+            _ => (),
```

Comment:
It should panic in this case as `panic!("code does not support {}", code.object_type)`